Sources/LeafKit/Exports.swift

```diff
@@ -1,8 +1,85 @@
 @_exported import NIO
 
+/// Various helper identities for convenience
```

Comment:
General whitespace cleanup and grouping of computed properties by functional use